plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Core/Type.hs

```diff
@@ -10,6 +10,7 @@ module UntypedPlutusCore.Core.Type
     ( TPLC.UniOf
     , TPLC.Version (..)
     , Term (..)
+    , TermLike (..)
```

Comment:
Actually good catch. That was from when I was trying to use these `TermLike` in the parser but I've moved away from that. I'll remove it and remove the `hiding var`'s.

Reply:
> What's happening here? I was trying to work out why `var` has to be hidden in `Internal.hs` and I think it's because `TermLike` is now exported here. Is that needed somewhere else?
> 
> [Later: looking at some of the superseded comments from earlier it seems that that might eventually be needed to make things more polymorphic over PLC/PIR. Is that right?]

Yes, that was why.